### ISSUE TITLE
Report that service is not ready if the IP is empty

### ISSUE THREAD

**Issue by sc3d** [UnityTech/ads-infra-localdev#2]
95% of the cases that we see this, is because the "user was impatient" (not really) and we had no IPs as the containers were not up yet.

That happens if it's waiting on a disk to be provisioned for example. Just about waiting, so let's print a warning letting the user know of that.

```
  ads-infra-localdev git:(master) ./service-env.sh setup
net.ipv4.ip_forward = 1
Password:
add net 10.0.0.0: gateway 192.168.99.100
add net 172.17.0.0: gateway 192.168.99.100
Setting up COMET_COMET - <none>:8090
./service-env.sh: line 65: none: No such file or directory
Setting up KAFKA_KONTROL - <none>:8000
./service-env.sh: line 65: none: No such file or directory
Setting up KAFKA_KAKFA - <none>:9092
./service-env.sh: line 65: none: No such file or directory
Setting up KUBERNETES_HTTPS - 10.0.2.15:443
Setting up PIXY_PIXY - <none>:19091
./service-env.sh: line 65: none: No such file or directory
Setting up TEST_HTTP - 172.17.0.3:80
Setting up ZOOKEEPER_ZOOKEEPER - 172.17.0.4:2181
Setting up ZOOKEEPER_KONTROL - 172.17.0.4:8000
bash-3.2$
```

Should do for now. We could put built in wait and retry loops but let's start simple.

**Comment by asko** [comment]
ideal expected behavior is to wait and complete once containers are up, but in the meanwhile failing the script with a message ala "Env is not yet up" is a lot better than returning unusable env vars without a clear way to retry

**Comment by sc3d** [comment]
Yep, I wonder if that will do more harm than good in "hopeless cases" where a fundamental problem (bad manifest/image/code) would prevent the dev cluster from starting at all.

I'd assume devs are kubernetes-savvy (thanks to our training) and will be able to keep an eye on "kubectl get all" to figure out what's going on.

Then yes, the warning would hint for them to do that. But yeah, I think that we're talking about the same thing.
